.github/ISSUE_TEMPLATE/config.yml

```
@@ -1,14 +1 @@
-    about: Open an issue in refarch if the topic is related to the web documentation
-  - name: 💡 Discussions - Ideas
-    url: https://github.com/it-at-m/refarch-templates/discussions/categories/ideas
-    about: Feel free to share ideas for the project that are to vague to create a dedicated issue beforehand
-  - name: ⁉ Discussions - Q&A
-    url: https://github.com/it-at-m/refarch-templates/discussions/categories/q-a
-    about: Please ask questions here
-  - name: 🗨 Discussions - General
-    url: https://github.com/it-at-m/refarch-templates/discussions/categories/general
-    about: Discuss more general topics here
\ No newline at end of file
+blank_issues_enabled: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)